### ISSUE TITLE
 Error: found in Chart.yaml, but missing in charts/ directory: prometheus, grafana

### ISSUE THREAD

**Issue by gnoejuan** [eclipse-che/che#18673]
### Describe the bug

` Error: found in Chart.yaml, but missing in charts/ directory: prometheus, grafana`

I wanted to utilize the '[Host-Access](https://microk8s.io/docs/addon-host-access)' plug in from Microk8s. So I `chectl server:delete` and `chectl server:deployed`

`chectl server:deploy -a helm -p microk8s -b 10.0.1.1`

### Che version

 - [x  ] latest `chectl/7.23.1 linux-x64 node-v12.20.0`
 - [ ] nightly
 - [ ] other: please specify

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ x ] other: (please specify) Microk8s

### Installation method
   - [ x ] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ x ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

### Eclipse Che Logs

```log
2020-12-20T02:49:29.970Z Error: Error: Unable to execute helm command helm upgrade --install che --force --namespace che --set global.cheDomain=10.0.1.1 --set global.tls.useSelfSignedCerts=true --set global.ingressDomain=10.0.1.1 --set cheImage=quay.io/eclipse/che-server:7.23.1 --set che.disableProbes=false  -f /home/charlesrod/.cache/chectl/templates/kubernetes/helm/che/values/tls.yaml  /home/charlesrod/.cache/chectl/templates/kubernetes/helm/che/ / Error: found in Chart.yaml, but missing in charts/ directory: prometheus, grafana
2020-12-20T02:49:29.970Z Command server:deploy failed. Error log: /home/charlesrod/.cache/chectl/error.log
2020-12-20T02:49:29.970Z     at Object.error (~/.local/share/chectl/client/7.23.1/node_modules/@oclif/errors/lib/index.js:26:15)
2020-12-20T02:49:29.970Z     at Deploy.error (~/.local/share/chectl/client/7.23.1/node_modules/@oclif/command/lib/command.js:60:23)
2020-12-20T02:49:29.970Z     at Deploy.<anonymous> (~/.local/share/chectl/client/7.23.1/lib/commands/server/deploy.js:208:22)
2020-12-20T02:49:29.970Z     at Generator.throw (<anonymous>)
2020-12-20T02:49:29.970Z     at rejected (~/.local/share/chectl/client/7.23.1/node_modules/tslib/tslib.js:113:69)

```

### Additional context

```
charlesrod@E560:~$ helm version
version.BuildInfo{Version:"v3.4.2", GitCommit:"23dd3af5e19a02d4f4baa5b2f242645a1a3af629", GitTreeState:"clean", GoVersion:"go1.14.13"}

```

**Comment by tolusha** [comment]
Unfortunately all chectl versions are affected.
It is already fixed in the nightly version of chectl and will be fixed in chectl 7.23.2

Meanwhile update `$HOME/.cache/chectl/templates/kubernetes/helm/che/requirements.lock` file manually like here
https://github.com/eclipse/che/pull/18651/files

**Comment by gnoejuan** [comment]
Chectl undo's the manual update.

`codium .cache/chectl/templates/kubernetes/helm/che` 

requirements.lock

```
...
- name: prometheus
  repository: https://prometheus-community.github.io/helm-charts
  version: 9.7.5
- name: grafana
  repository: https://grafana.github.io/helm-charts
  version: 4.6.3
digest: sha256:0d4d6e521cc91b902bf69ba89b86d30b7e29ccb1258fd0ce64e6e0b4e6f50755
generated: "2020-12-21T01:55:03.362968817-06:00"
```

`chectl server:deploy -a helm -p microk8s `

requirements.lock reverts to [during deploy]

```
...
- name: prometheus
  repository: https://kubernetes-charts.storage.googleapis.com/
  version: 9.7.5
- name: grafana
  repository: https://kubernetes-charts.storage.googleapis.com/
  version: 4.6.3
digest: sha256:0d4d6e521cc91b902bf69ba89b86d30b7e29ccb1258fd0ce64e6e0b4e6f50755
generated: "2020-12-21T02:05:05.338945945-06:00"
```

**Comment by tolusha** [comment]
Pls update this one if exists
`$HOME/.local/share/chectl/client/7.23.2/templates/kubernetes/helm/che/requirements.lock`
else 
`/usr/local/lib/chectl/templates/kubernetes/helm/che/requirements.lock`

**Comment by gnoejuan** [comment]
I think that should be 7.23.1 since I'm not on nightly. 

Yes, that was it. 

`$HOME/.local/share/chectl/client/7.23.1/templates/kubernetes/helm/che/requirements.yaml`

there was no .lock file.

I believe this issue is resolved and can be closed since this will be updated in `7.23.2`.
